src/main/java/gyro/google/storage/BucketResource.java

```diff
@@ -77,6 +78,27 @@
  *              end
  *          end
  *
+ *          iam-policy
+ *              bindings
+ *                  role: 'roles/storage.legacyBucketOwner'
+ *                  members: ['projectEditor:<<PROJECT-ID-GOES-HERE>>']
```

Comment:
```suggestion
 *                  members: ['projectEditor:<<PROJECT-ID-GOES-HERE>>'/the name of the role]
```

Reply:
When using `projectEditor`, the member value should just be the project id.